content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.OI
+  - ib-19.3.Datatype-Primitief-Integer.OI
+---
+
+## Wat is een Integer?
+Een Integer is een datatype waar een positief of negatief getal in past. Een programmeur kan dit datatype aan een variabel geven en hiermee getallen bewaren, bewerken en uitlezen.
```

Comment:
Opmerking: volgens mij is het een 'variabele', en is het bijbehorende lidwoord 'de'. Dus 'de variabele'. Variabel is een bijvoeglijk naamwoord. Deze zie ik vaker terug in de docs, dus wel zo net om even aan te passen :)

Note: jajaaaa, 'de variabele' is ook legit om als bvnw te gebruiken, maar dan heb je het niet over 'een' variabele :P